.yarn/versions/3fbef0f1.yml

```diff
@@ -1,4 +1,9 @@
 releases:
+  "@moonrepo/nx-compat": minor
```

Comment:
I don't release these unless required, you can mark them as declined like this: https://github.com/moonrepo/moon/blob/master/.yarn/declined.txt

Reply:
Just wanted that the pipeline will be executed successfully, I have deleted this file and let the release versioning on your side.

Reply:
Rebased to current develop-1.30